lib/run-in-terminal.coffee

```diff
@@ -87,30 +87,32 @@ start_terminal = (start_path, args) =>
 
             shebang = read_shebang(file_path)
 
-        for pair in read_option("launchers").split(",").map(strip)
+        for pair in read_option("programs").split(",").map(strip)
```

Comment:
I reversed the change, however since more options have changed it may be necessary to update to 1.0.0 anyway?